src/components/Loading/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {Text, View} from 'react-native';
+
+import {styles} from '~/components/Loading/styles';
+
+export default Loading = ({ loading, error, weather, temperature, location }) => {
```

Comment:
this component doesn’t looks like loading component
it looks like error component